media-sound/drumkv1/drumkv1-9999.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="An old-school all-digital drum-kit sampler synthesizer with stereo fx"
+HOMEPAGE="http://drumkv1.sourceforge.net/"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://download.sourceforge.net/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="mirror"
+
+IUSE="debug -qt4 +standalone alsa +jackmidi jacksession lv2 osc"
+REQUIRED_USE="
+	|| ( standalone lv2 )
+	standalone? ( || ( alsa jackmidi ) )
+	jackmidi? ( standalone )
+	jacksession? ( standalone )"
+
+RDEPEND="
+	qt4? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
+	!qt4? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtxml:5
+	)
+	media-libs/libsndfile
+	standalone? ( virtual/jack )
+	alsa? ( media-libs/alsa-lib )
+	lv2? ( media-libs/lv2 )
+	osc? ( media-libs/liblo )
+"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	if [[ ${PV} == *9999 ]]; then
+		eautoreconf
+	fi
+
+	# Disable stripping
+	echo "QMAKE_STRIP=" >> src/src_core.pri.in
```

Comment:
Because stripping shouldn't be done during install, see

From https://www.gnu.org/prep/standards/html_node/Standard-Targets.html
> Do not strip executables when installing them. This helps eventual debugging that may be needed later, and nowadays disk space is cheap and dynamic loaders typically ensure debug sections are not loaded during normal execution.

This allows everyone to build a proper debug build if necessary, see https://wiki.gentoo.org/wiki/Debugging#Don.27t_strip_symbols